### PR TITLE
WIP: 2.3.0 Changelog Audit

### PR DESCRIPTION
**Pull request by walterroach**
I reviewed the 2.3 project for closed issues, check here to see if anything's been missed that should be included.

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -3,18 +3,33 @@
...
 * **[Flight Planner]** Added BAI missions.
 * **[Flight Planner]** Added anti-ship missions.
 * **[Flight Planner]** Differentiated BARCAP and TARCAP. TARCAP is now for hostile areas and will arrive before the package.
+* **[Flight Planner]** Added OCA missions
+* **[Flight Planner]** Added Alternate/divert airfields
 * **[Culling]** Added possibility to include/exclude carriers from culling zones
 * **[QOL]** On liberation startup, your latest save game is loaded automatically
+* **[Units]** Reduced starting fuel load for C101
+* **[UI]** Inform the user of the weather
+* **[Factions]** Added livery overrides in faction files
```
Comment (Khopa) [inline changelog.md]: This was already in 2.2.1

**Review by Khopa — CHANGES_REQUESTED** [review]
The livery override settings was already added in 2.2.1; unless you mean we added new ones ?

Otherwise all good for me.  👍

File: changelog.md [hunk 2]
```diff
@@ -3,18 +3,33 @@
...
+* **[Game]** Added new Economy options for adjusting income multipliers and starting budgets.
 
 ## Fixes :
```
Comment (DanAlbert) [inline changelog.md:26]: Opfor now obeying parking limits seems worth mentioning.

File: changelog.md [hunk 3]
```diff
@@ -3,18 +3,33 @@
 # Features/Improvements
```
Comment (DanAlbert) [inline changelog.md:3]: Worth mentioning the ongoing l10n/i18n? It's a pretty small number of areas atm, but folks would probably be glad to hear that it's at least in progress.

**Review by DanAlbert — APPROVED** [review]
Otherwise looks good. Thanks for taking the time to clean up!

File: changelog.md (hunk 3 above)
Comment (DanAlbert) [inline changelog.md:3]: I just snuck in another worth mentioning: New Campaign: Black Sea

Comment (walterroach) [reply changelog.md:3]: Locale selection isn't in the dev branch, so there's nothing actually accessible to the user that's translated.  

I could add the selection back in if we want to, than the Chinese briefing we have would be accessible. The Enums I implemented are getting replaced with built-in language types, so I closed the original PR without merge.

Comment (DanAlbert) [inline changelog.md:3]:
```
* **[Mission Generator]** Mission targets and aircraft will no longer be culled. See [the manual](https://github.com/Khopa/dcs_liberation/wiki/Performance-Options#distant-unit-culling) for more information.
```

(didn't want to add this to `develop` and give you a nasty merge to resolve)

**Pull request merged**